lib/Class/Method/Modifiers.pm

```diff
@@ -18,8 +18,15 @@ our %EXPORT_TAGS = (
 
 BEGIN {
   *_HAS_READONLY = $] >= 5.008 ? sub(){1} : sub(){0};
+  *_MRO_MODULE   = "$]" < 5.010 ? sub() { "MRO/Compat.pm" } : sub() { "mro.pm" };
```

Comment:
We'd definitely want to go with the variant of this that doesn't require MRO::Compat.